sites/all/modules/publizon/publizon.module

```diff
@@ -363,13 +363,13 @@ function publizon_form_ding_reservation_reserve_form_alter(array &$form, array &
   $form['publizon']['phone_notification'] = array(
     '#type' => 'checkbox',
     '#title' => t('Send notification to my phone'),
-    '#default_value' => TRUE,
+    '#default_value' => FALSE,
     '#weight' => 0,
   );
   // Only show this field when the 'phone_notification' checkbox is enabled.
   $form['publizon']['field_phone']['#states'] = array(
     'visible' => array(
-      ':input[name="publizon[phone_notification]"]' => array('checked' => TRUE),
+      ':input[name="publizon[phone_notification]"]' => array('checked' => FALSE),
```

Comment:
This should not be changed. This is a [Conditional Form Field](https://www.drupal.org/docs/drupal-apis/form-api/conditional-form-fields) and the field `field_phone` should be `visible` when `phone_notification` is checked, i.e. when `'checked'` is `TRUE` (cf. https://www.drupal.org/docs/drupal-apis/form-api/conditional-form-fields#s-states-based-on-boolean-form-fields-checkboxes)

```suggestion
      ':input[name="publizon[phone_notification]"]' => array('checked' => TRUE),
```